Packages/com.unity.inputsystem/InputSystem/Plugins/iOS/iOSScreenKeyboard.mm

```diff
@@ -0,0 +1,651 @@
+#include "iOSScreenKeyboardBridge.h"
+#include "DisplayManager.h"
+#include "UnityAppController.h"
+#include "UnityForwardDecls.h"
+#include <string>
+
+#define KEYBOARD_LOG(...) LoggingIndentation s_IndentationScope; if (s_KeyboardLogging) NSLog(@"ScreenKeyboard -%@%@", LoggingIndentation::GetIndentation(), [NSString stringWithFormat: __VA_ARGS__])
```

Comment:
I don't like this macro for two issues it has:
1. It's function-like, but you can only call it once per scope due to variable definition. (And why variable is like static?)
2. It has potentially harmful if at the end

Suggestion:
- Make variable unique for each call, something like: LoggingIndentation indentationScope ## __LINE__
- Pass __FUNCTION__ to scope constructor and indent all same function logs at the same level
- Wrap if in a do { } while(0)